src/args.rs

```diff
@@ -75,7 +75,7 @@ pub struct InitArgs {
 }
 impl InitArgs {
     pub fn get_target_language(&self) -> TargetLanguage {
-        self.language.unwrap_or_else(|| {
+        self.language.unwrap_or({
```

Comment:
Just a minor thing but `unwrap_or_else` is more appropriate here, because `unwrap_or` evaluates the argument eagerly.

Reply:
Thanks. I added this change to my PR